test/e2e/change_rule_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
Can we change the year to 2024 and use Carvel Authors instead of VMware now?

Reply:
This should cause a CI failure now that the headers have been updated in https://github.com/carvel-dev/kapp/pull/903

Might need to do a rebase to properly pick up the changes and make CI detect it

Reply:
Done